xml/System.Windows.Forms/TaskDialogProgressBarState.xml

```diff
@@ -1,154 +1,182 @@
+      <MemberSignature Language="C++ CLI" Value="MarqueePaused" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.TaskDialogProgressBarState</ReturnType>
+      </ReturnValue>
+      <MemberValue>4</MemberValue>
+      <Docs>
+        <summary>
+          <para>Shows a marquee progress bar where the marquee animation is paused.</para>
+          <para>If you switch from <see cref="T:System.Windows.Forms.TaskDialogProgressBarState.Marquee" /> to <see cref="T:System.Windows.Forms.TaskDialogProgressBarState.MarqueePaused" /> while the dialog is shown, the marquee animation will stop.</para>
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+
+              If you switch from <xref:System.Windows.Forms.TaskDialogProgressBarState.Marquee> to <xref:System.Windows.Forms.TaskDialogProgressBarState.MarqueePaused> while the
+              dialog is shown, the  marquee animation will stop.
+            
+
+
+
+          ]]></format>
+        </remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="None">
+      <MemberSignature Language="C#" Value="None" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Windows.Forms.TaskDialogProgressBarState None = int32(5)" />
+      <MemberSignature Language="DocId" Value="F:System.Windows.Forms.TaskDialogProgressBarState.None" />
+      <MemberSignature Language="VB.NET" Value="None" />
+      <MemberSignature Language="F#" Value="None = 5" Usage="System.Windows.Forms.TaskDialogProgressBarState.None" />
+      <MemberSignature Language="C++ CLI" Value="None" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.TaskDialogProgressBarState</ReturnType>
+      </ReturnValue>
+      <MemberValue>5</MemberValue>
+      <Docs>
+        <summary>
+          <para>The progress bar will not be displayed.</para>
+          <para>While the dialog is showing, you cannot switch from <see cref="F:System.Windows.Forms.TaskDialogProgressBarState.None" /> to any other state, and vice versa.</para>
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+
+              Note: While the dialog is showing, you cannot switch from
+              <xref:System.Windows.Forms.TaskDialogProgressBarState.None> to any other state, and vice versa.
+            

```

Comment:
The preview doesn't have "Note:" 
![image](https://user-images.githubusercontent.com/4403806/127412098-00e6cc6e-9f4e-47a3-a13e-1a656c6c21ba.png)

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.taskdialogprogressbarstate?view=net-6.0&branch=pr-en-us-6940#fields

And even if it did, it'd look redundant
```suggestion
              While the dialog is showing, you cannot switch from
              <xref:System.Windows.Forms.TaskDialogProgressBarState.None> to any other state, and vice versa.
            

```